WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginSiteAddressFragment.java

```diff
@@ -198,7 +208,42 @@ protected void discover() {
         }
 
         mRequestedSiteAddress = getCleanedSiteAddress();
-        mDispatcher.dispatch(AuthenticationActionBuilder.newDiscoverEndpointAction(mRequestedSiteAddress));
+
+        Uri uri = Uri.parse(UrlUtils.addUrlSchemeIfNeeded(mRequestedSiteAddress, false));
+        String endpoint = WPCOMREST.sites.getUrlV1() + uri.getHost();
+        WPComGsonRequest req = WPComGsonRequest.buildGetRequest(endpoint, null, SiteWPComRestResponse.class,
+                new Response.Listener<SiteWPComRestResponse>() {
+                    @Override
+                    public void onResponse(SiteWPComRestResponse response) {
+                        if (response.jetpack) {
```

Comment:
I have a Jetpack site at http://www.nbradbury.com/wp/ but it's not getting handled here, instead it gets caught by `onErrorResponse` below (`apiError = `unknown_blog`). Is this expected?